src/Security/Security/Cmdlets/SqlVulnerabilityAssessment/RemoveSqlVulnerabilityAssessmentBaselineRule.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.Azure.Commands.Security.Common;
+using Microsoft.Azure.Commands.SecurityCenter.Models.SqlVulnerabilityAssessment;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.SecurityCenter.Cmdlets.SqlVulnerabilityAssessment
+{
+    [Cmdlet(VerbsCommon.Remove, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlVulnerabilityAssessmentBaselineRule", DefaultParameterSetName = ParameterSetNames.ResourceId, SupportsShouldProcess = true), OutputType(typeof(bool))]
+    public class RemoveSqlVulnerabilityAssessmentBaselineRule : SqlVulnerabilityAssessmentCmdlet
+    {
+        [Parameter(ParameterSetName = ParameterSetNames.ResourceId, Mandatory = true, HelpMessage = ParameterHelpMessages.RuleId)]
+        [Parameter(ParameterSetName = ParameterSetNames.OnPremMachines, Mandatory = true, HelpMessage = ParameterHelpMessages.RuleId)]
+        public string RuleId { get; set; }
+
+        [Parameter(ParameterSetName = ParameterSetNames.InputObject, Mandatory = true, ValueFromPipeline = true, HelpMessage = ParameterHelpMessages.InputObject)]
```

Comment:
Please add -Force and -PassThru as parameter

Reply:
pending